src/elements/ProjectTable/ProjectTable.js

```diff
@@ -58,9 +58,11 @@ const ProjectTable = ({ params, table }) => {
                       !Array.isArray(body[key].value) &&
                       `status_${body[key].value.toLowerCase()} capitalize`
                   )
-                  const name = body[key].value.startsWith(params.projectName)
-                    ? body[key].value.slice(params.projectName.length + 1)
-                    : body[key].value
+
+                  const extractedProjectName =
```

Comment:
I think we can get name right after {table.body.map((body, index) => {
and looks like extractedProjectName is not suitable variable name because it is item name without "project name"